### DISCUSSION TITLE
Can sleap detect the different colors?

### DISCUSSION THREAD

**Discussion by miraclezero** [talmolab/sleap#1355] [Help!]
If I have two identical objects with two different colors, can sleap detect them based on their colors?
Thanks

**Comment by roomrys** [comment]
Hi @miraclezero,

SLEAP will use the pixel values of multiple annotations to determine what a feature looks like and how to find that feature in new images. SLEAP does this through more than just the color, but also the value of pixels surrounding the feature of interest. SLEAP does not do segmentation (or classification based on color), but it will be able to distinguish between features of different colors.

Let us know if that helps or if you have more questions!

Thanks,
Liezl
